container builds for flavor reservation

msherman64 opened the pull request:
this PR builds containers for the 2023.1 flavor reservation on kvm case.

it cannot be merged as-is, because we still need to build two separate "sets" of horizon containers for baremetal + kvm

This builds the changes pulled in from:
- https://github.com/ChameleonCloud/blazar/pull/123
- https://github.com/ChameleonCloud/blazar-dashboard/pull/44
- https://github.com/ChameleonCloud/horizon/pull/28

@Mark-Powers are we able to merge the blazar and blazar-dashboard ones? I wasn't sure if they had conflicts too.

msherman64 committed:
pin kolla commit for ussuri

msherman64 committed:
remove incompatible configs

msherman64 committed:
make sure we build source version

msherman64 committed:
container builds for flavor reservation

Mark-Powers commented:
@msherman64 No conflicts in merging blazar or blazar-dashboard, but we should run our tests and double check on UC dev.

msherman64 committed:
update refs for victoria

msherman64 committed:
update refs for wallaby

msherman64 committed:
update refs for xena

msherman64 committed:
bump to yoga

msherman64 committed:
bump refs to zed

msherman64 committed:
bump refs for 2023.1

msherman64 committed:
bump ref for letsencyrpt

msherman64 committed:
ensure fork of blazarclient is installed

Mark-Powers committed:
Add support for flavor leases (#52)

* Remove blazarclient #egg requirement

This is breaking CI, and is discouraged by the pip documentation:
- https://pip.pypa.io/en/stable/topics/vcs-support/#url-fragments

* Include servers in lease reminder email (#50)

* Include servers in lease reminder email

* Add check for servers

* Install chameleon-blazarclient in blazar-manager

* Add check for KVM when getting servers

* Add support for flavor leases

* Fix KVM server lookup in lease email

* Update blazarclient dependency specification

pdmars committed:
bump with fix for pip version (#54)

Co-authored-by: Michael Sherman <shermanm@uchicago.edu>
